.github/stale_issue.yaml

```diff
@@ -0,0 +1,45 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+on:
+  schedule:
+  - cron: "*/15 * * * *"
+
+jobs:
+  cleanup:
```

Comment:
Nitpick: `cleanup` job id is a little ambiguous here. Maybe rename it to `issue-cleanup`?